Functions.Templates/Templates/HttpTrigger-CSharp-Isolated-NetFx/.template.config/template.json

```diff
@@ -0,0 +1,84 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+        "Http"
+    ],
+    "name": "HttpTrigger",
+    "identity": "Azure.Function.CSharp.Isolated.NetFx.HttpTrigger.1.x",
+    "groupIdentity": "Azure.Function.HttpTrigger",
+    "shortName": "http",
```

Comment:
note: will need a different short name

Reply:
Closing - only need different short name if there are two triggers in one package but we are going with splitting.

Reply:
Jk core-tools will ship both packages together so changing this

Reply:
Wait, so what happens if someone wants to just use the `dotnet` CLI? I guess they too would install the netfx version, but then how do they tell the system which to use? Am I correct in assuming that the Core Tools is only doing the changeup for HTTP, or would we do it across the board for all templates?

Reply:
Given that the rest (not http trigger) of the templates are same, it might be okay to pick either one. But for http we do need a different short name if both the packages are to be installed and used at the same time. I am not sure how dotnet CLI handles the cache for two templates with same short name. We would have to test this out to see how it works.